### ISSUE TITLE
[BUG] iroha_client 401 HTTP Response after executing get_config_value method

### ISSUE THREAD

**Issue by Stukalov-A-M** [hyperledger/iroha#4177]
It happens because method doesn't pass Authorization header to the Request builder.
~~Also we are using both attohttpc and http crates for building HTTP requests, that creates code excessiveness and in some cases ambiguousness.
As an example - building a request through request body check.~~
```
pub fn build(self) -> Result<DefaultRequest> {
        self.inner
            .map(|b| DefaultRequest(b.bytes(self.body.map_or_else(Vec::new, |vec| vec))))
    }
```
~~For this reason, in the process of the bug fixing, I propose to refactor by removing the attohttpc crate from client and unify all HTTP request building using the http crate only.~~

**Comment by Stukalov-A-M** [comment]
Discussed with Daniil and Dmitriy that it's not safety to refactor request builder because current design provides a generic property for other request builders like ActixReqBuilder.
So, it was decided to add Authorization header only into "client/get_config()"

**Comment by nxsaken** [comment]
Fixed by #4180.
